scrowl-io/scrowl
# sc-1047 - Feature -Open a Project

### Short summary

As a course creator, I want to be able to open existing course projects so that I can make changes and also publish the course.

When you select File->Open... on the top menu, the Windows or Mac's "Open dialog window" should pop up to select the Project to be opened.

---

### Test steps
1. Run `yarn start:authoring`
2. Using the app toolbar go to `File > New Project...` to create a new project
3. In the app toolbar go to `File > Save` to save the project in a `.scrowl` file
4. Restart the application, go to `File > Open...`, and select the project you saved in the previous step.
5. The project should be loaded, and saved into the project model. To check out the project data you can add a `console.log` with the project data in the `apps/authoring/src/renderer/pages/home/page-home.tsx` component file.
